android/src/main/java/com/github/douglasjunior/reactNativePdfRenderer/modules/PdfRendererRecyclerView.java

```diff
@@ -419,8 +419,8 @@ public ImageView getImageView() {
             }
 
             private @NonNull Bitmap createBitmap(float newZoom, int pageWidth, int pageHeight) {
-                var scaledPageWidth = Math.round(pageWidth * newZoom);
-                var scaledPageHeight = Math.round(pageHeight * newZoom);
+                var scaledPageWidth = Math.round(getResources().getDisplayMetrics().densityDpi * pageWidth * newZoom / 72);
```

Comment:
Please, just add a comment with the link for the explanation to this formula.

Example:

```suggestion
                // Calculate the size to match PDF rendering quality with a high-DPI screen 
                // https://stackoverflow.com/a/32327174/2826279
                var scaledPageWidth = Math.round(getResources().getDisplayMetrics().densityDpi * pageWidth * newZoom / 72);
```